app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = current_user.comments.new(comment_params)
+    @comment.post_id = @post.id
+    @comment.save
+    redirect_to "/users/#{@post.user_id}/posts/#{@post.id}"
+  end
+
```

Comment:
Hello @enaburekhan, with the way my code is currently structured a `new` method doesn't serve any purpose. It was left out deliberately because of this reason.

Reply:
This change has been done well